vernac/declare.ml

```diff
@@ -1921,10 +1921,13 @@ end = struct
     in
     List.map_i (declare_mutdef ~pinfo ~uctx pe) 0 pinfo.Proof_info.cinfo
 
+  let update_mutual_pe ~typ (v,(_,univs),i) = (v,(typ,univs),i)
```

Comment:
Univs should be dropped if monomorphic, same as next_univs in comassumption

Reply:
Should we add a test case for the polmorphic case?